server/controllers/bookingControllers.js

```diff
@@ -17,42 +25,50 @@ class bookingsController {
 
   // // ###check trip ID
   const tripid = req.body.tripId;
-  const findtripid = trips.find(findtripid => findtripid.id == tripid);
-  if (!findtripid) 
- { 
-   
-  return response.response(res, 404,'error', 'No trip found!', true);
-}
-  // check if trip is activated*
+  let findtripid = await client.query('SELECT * FROM trips WHERE id=$1',[
+    req.body.tripId,
```

Comment:
You just declared a variable tripid from `req.body.tripId`, why not use that?

Reply:
I forget to remove it